app/models/search.rb

```diff
@@ -62,6 +70,27 @@ def start! search_id
       search.pseudo_graph_pattern.update(started_at: Time.now.utc)
       search
     end
+
+    private
+
+    # rubocop:disable Metrics/AbcSize
+    def hash_dialog_search search
+      {
+        search_id: search.search_id, query: search.query,
+        read_timeout: search.pseudo_graph_pattern.read_timeout,
```

Comment:
```
pgp = search.pseudo_graph_pattern
```
みたいに短い名前のローカル変数に代入してあげると

```suggestion
        read_timeout: pgp.read_timeout,
```

となって、読みやすくなるとおもいます。

あるいは
```
 delegate :read_timeout, to: :pseudo_graph_pattern
```
https://apidock.com/rails/Module/delegate

としてしまうのも手かもしれません。

Reply:
対応しました。